docs/topics/polygon/polygon.md

```diff
@@ -0,0 +1,147 @@
+| RootChainManager                                                                              | The RootChainManager is a contract deployed by Polygon on L1. Its purpose is to trigger the state sync mechanism of polygon after depositing an asset on the predicate contract on L1.                                                                                                                                                                                                                                                                                                                                                                                                                         |
+| ChildChainManager                                                                             | The ChildChainManager is a contract on L2 triggered by the a call on the RootChainManager on L1 through the state sync mechanism. Then, the ChildChainManager calls the deposit function of the child token contract to mint the tokens on L2.                                                                                                                                                                                                                                                                                                                                                                 |
+| Checkpoint                                                                                    | All transactions that occur on L2 are check-pointed to L1 in frequent intervals of time by the validators.                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                     |
+| State sync mechanism                                                                          | The state sync is an internal mechanism of Polygon to trigger from a call to the RootChainManager on L1, the ChildChainManager contract on L2.                                                                     |
+
+## Transferring assets from L1 to L2
+
+When the owner of a token wants to use his token on L2 (Polygon chain), a process of locking tokens on a contract provided by Polygon on L1 called a predicate is necessary.  
```

Comment:
-wouldn't it be better to say "send" instead of "use"?

-a process of locking tokens on a predicate contract (provided by Polygon on L1)  is necessary.